olca-app/plugin.xml

```diff
@@ -610,6 +610,10 @@
 					name="startupMessageRect"
 					value="7,525,445,20">
 			</property>
+   <property
+         name="preferenceCustomization"
+         value="plugin_customization.ini">
+   </property>
```

Comment:
Hi @msrocka. Could you help me understand if this change is safe? This was the result of changing the icons but I am not sure why or how.

Reply:
Yes, this is generated automatically from Eclipse and not necessary. You can just reset that file.